src/main/java/org/damap/base/conversion/AbstractTemplateExportSetup.java

```diff
@@ -38,7 +38,7 @@ public abstract class AbstractTemplateExportSetup extends AbstractTemplateExport
   protected Properties prop = null;
   protected List<XWPFParagraph> xwpfParagraphs = null;
   protected List<XWPFTable> xwpfTables = null;
-  protected ContributorDO projectCoordinator = null;
+  protected List<ContributorDO> projectCoordinator = new ArrayList<>();
```

Comment:
This list should be renamed to projectCoordinators, since there can be multiple coordinators.

Reply:
Agree